### PR TITLE
Trigger homebrew update on each release.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: .github/workflows/dispatch_release.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Update homebrew
+
+on:
+  release:
+    types: [ published ]
```
Comment (kasperl) [inline .github/workflows/dispatch_release.yml:5]: Does this fire for pre-releases too? Not sure this is a good idea. It feels hard to go back if a release turns out to be broken.

Comment (floitsch) [reply .github/workflows/dispatch_release.yml]: Changed to "released". Hopefully that's what we want.

File: .github/workflows/dispatch_release.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Update homebrew
+
+on:
+  release:
```
Comment (kasperl) [inline .github/workflows/dispatch_release.yml:4]: Can we check that we don't do this for releases marked as 'pre-release'?

Comment (floitsch) [reply .github/workflows/dispatch_release.yml:4]: In theory that's what "released" is for: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#release

**Review by kasperl — APPROVED** [review]

**Pull request merged**